alerta-covid/app/controllers/api/v1/institutions_controller.rb

```diff
@@ -4,57 +4,45 @@ class Api::V1::InstitutionsController < ApplicationController
   
     # GET api/v1/institutions or /institutions.json
     def index
-      institutions = Institution.all
-      render json: {status: 200, data: institutions} 
+      @institutions = Institution.all
+      render json: @institutions, status: :ok
     end
   
     # GET api/v1/institutions/1 or /institutions/1.json
     def show
-      render json: {status: 200, data: @institution}
+      render json: @institution, status: :ok
     end
   
     # POST api/v1/institutions or /institutions.json
     def create
-      begin
-        @institution = Institution.new(institution_params)
-    
-        if @institution.save
-          render json: {status: 200, data: @institution }
-        else
-          render json: {msg: @institution.errors, status: 500}
-        end 
-      rescue
-        render json: {msg: :server_error, status: 500}
-      end
+      @institution = Institution.new(institution_params)
+
+      if @institution.savew
```

Comment:
Typo

Would be nice if you add some unitesting to prevent publishing code with typos 😄 
That way you ensure it works without having to pass by manual testing